Sources/SparkConnect/DataFrame.swift

```diff
@@ -23,7 +23,158 @@ import GRPCNIOTransportHTTP2
+///
+/// A DataFrame is equivalent to a relational table in Spark SQL, and can be created using various
+/// functions in ``SparkSession``. Once created, it can be manipulated using the various domain-specific
+/// language (DSL) functions defined in: ``DataFrame``, ``Column``, and functions.
+///
+/// ## Creating DataFrames
+///
+/// DataFrames can be created from various sources:
+///
+/// ```swift
+/// // From a range
+/// let df1 = try await spark.range(1, 100)
+///
+/// // From a SQL query
+/// let df2 = try await spark.sql("SELECT * FROM users")
+///
+/// // From files
+/// let df3 = try await spark.read.csv("data.csv")
+/// ```
+///
+/// ## Common Operations
+///
+/// ### Transformations
+///
+/// ```swift
+/// // Select specific columns
+/// let names = try await df.select("name", "age")
+///
+/// // Filter rows
+/// let adults = try await df.filter("age >= 18")
+///
+/// // Group and aggregate
+/// let stats = try await df.groupBy("department").agg("avg(salary)", "count(*)")
+/// ```
+///
+/// ### Actions
+///
+/// ```swift
+/// // Show the first 20 rows
+/// try await df.show()
+///
+/// // Collect all data to the driver
+/// let rows = try await df.collect()
+///
+/// // Count rows
+/// let count = try await df.count()
+/// ```
+///
+/// ## Topics
+///
+/// ### Basic Information
+/// - ``columns``
+/// - ``schema``
+/// - ``dtypes``
+/// - ``sparkSession``
+///
+/// ### Data Collection
+/// - ``count()``
+/// - ``collect()``
+/// - ``head(_:)``
+/// - ``tail(_:)``
+/// - ``show()``
+/// - ``show(_:)``
+/// - ``show(_:_:)``
+/// - ``show(_:_:_:)``
+///
+/// ### Transformation Operations
+/// - ``select(_:)``
+/// - ``selectExpr(_:)``
+/// - ``filter(_:)``
+/// - ``where(_:)``
+/// - ``sort(_:)``
+/// - ``orderBy(_:)``
+/// - ``limit(_:)``
+/// - ``offset(_:)``
+/// - ``drop(_:)``
+/// - ``withColumnRenamed(_:_:)``
+///
+/// ### Join Operations
+/// - ``join(_:)``
+/// - ``join(_:_:_:)``
+/// - ``join(_:_:_:)-3b88j``
+/// - ``join(_:joinExprs:)``
+/// - ``join(_:joinExprs:joinType:)``
+/// - ``crossJoin(_:)``
+/// - ``lateralJoin(_:)``
+/// - ``lateralJoin(_:joinType:)``
+/// - ``lateralJoin(_:joinExprs:)``
+/// - ``lateralJoin(_:joinExprs:joinType:)``
+///
+/// ### Set Operations
+/// - ``union(_:)``
+/// - ``unionAll(_:)``
+/// - ``unionByName(_:_:)``
+/// - ``intersect(_:)``
+/// - ``intersectAll(_:)``
+/// - ``except(_:)``
+/// - ``exceptAll(_:)``
+///
+/// ### Partitioning
+/// - ``repartition(_:)``
+/// - ``repartition(_:)-5wxc7``
+/// - ``repartition(_:_:)``
+/// - ``repartitionByExpression(_:_:)``
+/// - ``coalesce(_:)``
+///
+/// ### Grouping Operations
+/// - ``groupBy(_:)``
+/// - ``rollup(_:)``
+/// - ``cube(_:)``
+///
+/// ### Persistence
+/// - ``cache()``
+/// - ``persist(storageLevel:)``
+/// - ``unpersist(blocking:)``
+/// - ``storageLevel``
+///
+/// ### Schema Information
+/// - ``printSchema()``
+/// - ``printSchema(_:)``
+/// - ``explain()``
+/// - ``explain(_:)``
+/// - ``explain(_:)-6r4o9``
```

Comment:
Hmm, what does this string `-6r4o9` mean?

Reply:
Let me remove that. It was added due to overloaded method.